examples/api/lib/widgets/overscroll_indicator/glowing_overscroll_indicator.2.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for GlowingOverscrollIndicator
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/material.dart';
+
+// Override the [default drag behavior](https://flutter.dev/docs/release/breaking-changes/default-scroll-behavior-drag).
```

Comment:
This isn't a helpful comment, since it just reiterates what's obvious. It would be helpful to note that we're adding support for PointerDeviceKind.mouse and it would be a little more correct to add that to super.dragDevices.

The API doc for dragDeviceProperties include a rather dire warning:

"Enabling this for PointerDeviceKind.mouse will make it difficult or impossible to select text in scrollable containers and is not recommended."

Is this example still advisable given the warning?

Reply:
That makes sense, noted.